src/static/js/ours/utils.js

```diff
@@ -117,14 +114,36 @@ function set_form_data(data, base_element) {
     var fields = get_form_fields(base_element)
     fields.each(function (i, field) {
         if (!!field.name) {
-            //console.log("@@@@@")
-            //console.log(field)
-            console.log(field.name + " -> " + data[field.name])
             $(field).val(data[field.name])
         }
     })
 }
 
+function objectToFormData(obj, form, namespace) {
+    var fd = form || new FormData()
+    var formKey
+
+    for (var property in obj) {
```

Comment:
having trouble wrapping my head around this entirely (it has been a long day...) but could we be doing this with something like `_.reduce`? this kinda seems like a reduce usecase? or am I confused? would be very on board with some tag-team code golfing this. (or maybe just an explanation of what I'm looking at....)

Reply:
I don't know about a `_.reduce` but to be clear: this is from a gist for turning a nested object into a `FormData` object